test_runner/fixtures/neon_fixtures.py

```diff
@@ -3158,6 +3158,67 @@ def __exit__(
         pass
 
 
+class RemoteNeonApiEndpoint:
+    def __init__(self, neon_api: NeonAPI, pg_version: PgVersion, project_id: Optional[str]):
+        self.neon_api = neon_api
+        if project_id is None:
+            project = neon_api.create_project(pg_version)
+            neon_api.wait_for_operation_to_finish(project["project"]["id"])
+            self.project_id = project["project"]["id"]
+            self.endpoint_id = project["endpoints"][0]["id"]
+            self.connstr = project["connection_uris"][0]["connection_uri"]
+            self.pgbench_env = connection_parameters_to_env(
+                project["connection_uris"][0]["connection_parameters"]
+            )
+            self.is_new = True
+        else:
+            project = neon_api.get_project_details(project_id)
+            if int(project["project"]["pg_version"]) != int(pg_version):
```

Comment:
Comparing ints probably doesn't matter here

Reply:
One is a string and one is a `pgversion`, so they have to get casted to the same type (probably doesn't matter exactly what that type is).